example-workflows/prep-release.yml

```diff
@@ -12,10 +12,10 @@ on:
       post_version_spec:
         description: "Post Version Specifier"
         required: false
-      # silent:
-      #   description: "Set a placeholder in the changelog and don't publish the release."
-      #   required: false
-      #   type: boolean
+      silent:
```

Comment:
iirc this was left commented so that the input does not show up in the UI as this is more of an advanced feature (discussed in https://github.com/jupyter-server/jupyter_releaser/pull/526)

Reply:
Okay fair, I'll comment it out